src/model/BaseEntity.ts

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
```

Comment:
i think this leads to a fully included lodash in production builds if i remember state of treeshaking correctly.
would be better to have an explicit import.

Reply:
Yes, you're right. Fixed in in fcd272b.